core/templates/domain/exploration/StatesObjectFactorySpec.ts

```diff
@@ -29,7 +29,9 @@ const constants = require('constants.ts');
 
 describe('States object factory', () => {
   describe('StatesObjectFactory', () => {
```

Comment:
Remove this describe block. There's no need to have two `describe` blocks nested with same name.

Reply:
Done